doc/releases/release-notes-3.3.rst

```diff
@@ -306,6 +306,8 @@ Boards & SoC Support
 * Changes for ARC boards:
 
 * Added support for these ARM boards:
+  * Adafruit KB2040
```

Comment:
This needs a blank line to render correctly (https://builds.zephyrproject.io/zephyr/pr/54636/docs/releases/release-notes-3.3.html#boards-soc-support)